### PR TITLE
VIH-8394 Added redirect to unauthorized page when feature toggled off

### PR DESCRIPTION
**Pull request by PrasannaDommalapati**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-8394

### Change description ###

- Toggled staffmember accessing hearing list after the login

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ZJianWang — DISMISSED** [review]

File: VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.spec.ts [hunk 1]
```diff
@@ -50,6 +61,19 @@ describe('NavigatorComponent', () => {
         expect(router.navigate).toHaveBeenCalledWith([pageUrls.JudgeHearingList]);
     });
 
+    it('should go to staff member hearing list if user is a StaffMember and staff-member-feature is ON', async () => {
+        featureFlagServiceSpy.getFeatureFlagByName.and.returnValue(of(false));
```
Comment (robertbennett1998) [inline VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.spec.ts:65]: Shouldn't this test fail? Returning false means the feature is turned off right?

Comment (robertbennett1998) [reply VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.spec.ts:65]: It looks like this should  be in the below test

**Review by robertbennett1998 — COMMENTED** [review]
Logic looks correct but the tests look like they should fail

**Review by robertbennett1998 — APPROVED** [review]
LGTM

File: VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.spec.ts (hunk 1 above)
Comment (PrasannaDommalapati) [reply VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.spec.ts]: fixed

**Review by ZJianWang — APPROVED** [review]

**Pull request merged**